backend/apps/slack/events/message_posted.py

```
@@ -39,10 +39,47 @@ def handle_event(self, event, client):
+                                    for text_element in element.get("elements", []):
+                                        if (
+                                            text_element.get("type") == "user"
+                                            and text_element.get("user_id") == bot_user_id
+                                        ):
+                                            bot_mentioned = True
+                                            break
+                                    if bot_mentioned:
+                                        break
+                            if bot_mentioned:
+                                break
+        except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)